mbw.c

```diff
@@ -57,6 +61,7 @@ void usage()
     printf("Usage: mbw [options] array_size_in_MiB\n");
     printf("Options:\n");
     printf("	-n: number of runs per test (0 to run forever)\n");
+    printf("	-w: number of workers (threads, default: %d)\n", DEFAULT_WORKERS);
```

Comment:
suggestion: mbw was written to simulate the use case of "naive" single-threaded code. While measuring hardware capabilities is also a great goal, why not default to the old behavior (1 thread)?